fineract-provider/src/main/java/org/apache/fineract/infrastructure/configuration/api/GlobalConfigurationApiResource.java

```diff
@@ -76,29 +76,44 @@ public GlobalConfigurationApiResource(final PlatformSecurityContext context,
     @GET
     @Consumes({ MediaType.APPLICATION_JSON })
     @Produces({ MediaType.APPLICATION_JSON })
-    public String retrieveConfiguration(@Context final UriInfo uriInfo,@DefaultValue("false") @QueryParam("survey") final boolean survey) {
+    public String retrieveConfiguration(@Context final UriInfo uriInfo,
```

Comment:
Hi Nazeer,
This is the new configuration: 
public String retrieveConfiguration(@Context final UriInfo uriInfo,@QueryParam("configName") final String configName,@DefaultValue("false") @QueryParam("survey") final boolean survey)


I have added configName as new parameter, beacuse we are going to fetch configuration by the name